dimscord/objects/macros.nim

```diff
@@ -60,3 +62,17 @@ macro optionIf*(check: typed): untyped =
 
     result = quote do:
         if `check`: none `varType` else: some (`variable`)
+
+macro dimClient*(x: typed): untyped =

```

Comment:
I was thinking of naming this to this instead
```suggestion
macro setHelper*(x: typed): untyped =
# Use this variable to use the helper functions. Can be set only once.
```

Reply:
I honestly think `dimClient` is fine, but i'll add the comments at least